src/hooks/useDispatch.js

```diff
@@ -1,4 +1,20 @@
-import { useStore } from './useStore'
+import { useStore as useDefaultStore, createStoreHook } from './useStore'
+
+/**
+ * Hook factory, which creates a `useDispatch` hook bound to a given context.
+ *
+ * @param {Function} [useReduxContext] Hook which returns the Redux context.
+ * @returns {Function} A `useDispatch` hook bound to the specified context.
+ */
+export function createDispatchHook(useReduxContext = null) {
```

Comment:
`useDispatch` relies on `useStore` for its implementation, not `useReduxContext`. I was tempted to use the signature `createDispatchHook(useStore)` instead of the current `createDispatchHook(useReduxContext)`, but I thought it might lead to user error since the remaining hook factories accept `useReduxContext`.